horovod/spark/common/params.py

```diff
@@ -72,10 +72,10 @@ class EstimatorParams(Params):
                         'random seed to use for DL frameworks',
                         typeConverter=TypeConverters.toInt)
 
-    shuffle_buffer_size = Param(Params._dummy(),
```

Comment:
This breaks existing user code, does it? Can we deprecate this field (print a warning) and ignore it? Or does the Horovod code get too messy then?

Reply:
I was aggressive on removing unnecessary fields. I will leave this field till next release, and throw out a deprecated warning.

Internally we don't need `shuffle_buffer_size ` for Petastorm 0.12.0.
Training samples are simply treated as shuffle or not.